src/frontend/apps/drive/package.json

```diff
@@ -10,7 +10,7 @@
     "build-theme": "cunningham -g css,scss -o src/styles && mv src/styles/cunningham-tokens.scss src/styles/cunningham-tokens-sass.scss"
   },
   "dependencies": {
-    "@gouvfr-lasuite/ui-kit": "0.4.0",
+    "@gouvfr-lasuite/ui-kit": "0.4.1",
```

Comment:
Nothing happens when clicking "..." on main workspace, let's not display this button ? 
<img width="377" alt="Capture d’écran 2025-04-25 à 14 37 30" src="https://github.com/user-attachments/assets/6019d872-13d6-4451-9b0a-d3ae26755d06" />